packages/api-client/src/views/Request/RequestSection/RequestTable.vue

```diff
@@ -78,16 +86,48 @@ const flattenValue = (item: RequestExampleParameter) => {
       v-for="(item, idx) in items"
       :key="idx">
       <label class="contents">
-        <span class="sr-only">Row Enabled</span>
-        <DataTableCheckbox
-          class="!border-r-1/2"
-          :disabled="hasCheckboxDisabled"
-          :modelValue="item.enabled"
-          @update:modelValue="(v) => emit('toggleRow', idx, v)" />
+        <template v-if="isGlobal">
+          <RouterLink
+            class="!border-r-1/2 border-t-1/2 text-c-2 flex justify-center items-center"
+            :to="item.route ?? {}">
+            <span class="sr-only">Global</span>
+            <ScalarTooltip
+              as="div"
+              class="z-[10001]"
```

Comment:
```suggestion
              class="z-[10001]"
```
do we need this class? @hwkr

Reply:
I don’t think so! Removed it.